content/faq/content.md

```diff
@@ -0,0 +1,34 @@
+---
+title: What kind of content can I upload, and what about illegal or infringing content?
+category: publisher
+---
+
+This guide provides answers about questions regarding what you may upload to the LBRY app and network, as well as how to report illegal or infringing content.
+
+###What content can I legally upload to LBRY?
+
+You may upload content you created or own that does not infringe on the rights or safety of others, or content you have permission to upload. This includes Public Domain content, Creative Commons or Open Source content shared according to the license requirements, and Copyrighted content where you have permission from the owner.
+
+###Isn’t LBRY decentralized and the content can’t be removed?
+
+It is important to make a distinction between the LBRY blockchain/protocol and any applications running on top of them when referring to censorship. The LBRY protocol is fully decentralized and censorship resistant, with the protocol storing metadata and naming on the blockchain, and facilitating data transfers over a peer to peer (P2P) network. However, LBRY also makes an app to demonstrate our protocol. With our app we will engage in non-arbitrary censorship, meaning only horrific or infringing content will be removed. As a U.S. company LBRY Inc. and our app will follow all U.S. laws including DMCA. If someone made an app or website using the LBRY protocol in some other country, it would have to follow that country's laws, which aren’t necessarily the same as ours. Either app would read the same blockchain, though. 
+
+###Someone uploaded my content to LBRY without permission. How do I remove it?
+
+Reporting infringing/illegal content in the LBRY App is easy. Open the LBRY App and navigate to the content you wish to report. Underneath the content are two buttons: "Download," and "Report." When you click on the Report button, you will be redirected to a web form to report the content to LBRY, Inc., who can remove the content link from the LBRY App. 
+
+###What do I do if I see content that’s illegal or infringing in the LBRY App?
+
+Please follow the FAQ above for reporting infringing content, and LBRY staff will review your report and take any appropriate action. 
+
+###What is a DMCA request or takedown notice?
+
+The Digital Millennium Copyright Act (DMCA) is United States Copyright Law written to protect digital content against illegal use. When an owner finds their content used on the Internet without their permission, they can file a DMCA request with the website, app, or Internet Service Provider illegally hosting their content, asking for it to be removed.
+
+###How does LBRY respond to a DMCA request or takedown notice?
+
+LBRY Inc., will remove links deemed infringing from the LBRY App when it receives a valid DMCA request from the content owner.
```

Comment:
Probably more accurate to say something like "LBRY, Inc. will block access to content deemed infringing..."

Possibly superfluous to say "from the content owner". As I understand it, DMCA requests are _only_ valid if from the owner.

Reply:
Possibly worth mentioning that we provided an API that can allow you to check if a specific piece of content is infringing according to our records.